fluid_onnx/ops.py

```diff
@@ -445,10 +447,10 @@ def xor_op():
 # ONNX Ops that use multiple Paddle ops are keyed by '<op1>,<op2>' fed into the
 # modifier.
 
-PADDLE_TO_ONNX = {
+node_maker = {
     # Paddle op name : (ONNX op name, modifier)
     'abs': ('Abs', abs_op),
-    'elementwise_add': ('Add', add_op),
+    'elementwise_add': add_op,
```

Comment:
So the reason I originally did this weird mapping to tuples is because we may want to reuse such a map in future for mapping the reverse way. So it will be easier to traverse that. If you think we will probably need to create an entirely way, this makes much more obvious sense. What do you think?

Reply:
I thought we can implement the bidirectional conversion in one map. But after I write some code, I feel it would be better to decouple the two conversions very clearly. Because it is a bit hard to make the two sets of operators one-to-one correspondence. For example, ONNX has `FC` operator, but Fluid doesn't, in Fluid--> ONNX conversion we would never use `FC` op, and in the reverse conversion we need to implement the `FC` op with `mul` and `elementwise_add` op in Fluid.

Reply:
Completely fair, let's go with this. I thought of the same, but yeah, I agree with your conclusion